code/modules/reagents/reagent_containers/food/cans.dm

```diff
@@ -333,7 +333,7 @@
 
 /obj/item/reagent_containers/food/drinks/cans/koispunch
 	name = "\improper Phoron Punch!"
-	desc = "A radical looking can of <span class='warning'>Phoron Punch!</span> Phoron poisoning has never been more extreme!"
+	desc = "A radical looking can of <span class='warning'>Phoron Punch!</span> Phoron poisoning has never been more extreme! <span class='danger'>WARNING: Phoron is toxic to humans. Consuming this product might lead to death.</span>"
```

Comment:
Generally you should avoid doing `<span class='warning'>` and `<span class='danger'>`. Instead it should be roughly:
`desc = "A radical looking can of " + SPAN_WARNING("Phoron Punch!") + " Phoron poisoning has never been more extreme! " + SPAN_DANGER("WARNING: Phoron is toxic to humans. Consuming this product might lead to death.")`